scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -160,6 +163,8 @@ private void releaseNodes(InternalJob job, NodeSet nodeSet) throws RMProxyCreati
      * @return the number of tasks that have been started
      */
     public int schedule() {
+        schedulingMainLoopTimingLogger = new SchedulingMainLoopTimingLogger(logger);
```

Comment:
the logger should be changed to SchedulingMethodImpl.class

Reply:
Why not passing the already created logger?